src/components/MobileImportExport/Export.tsx

```diff
@@ -44,6 +53,26 @@ export function ExportData() {
     }
     return (
         <main className={classes.root}>
+            <Typography style={{ marginBottom: 24 }}>
+                <Button style={{ marginRight: 12 }} size="small" variant="outlined" color="primary" disabled>
+                    Beta
+                </Button>
+                {'Checkout Maskbook '}
+                <Link target="_blank" href="https://testflight.apple.com/join/OGmGmIg1">
+                    iOS
+                </Link>
+                {' and '}
```

Comment:
Too many separate strings here. I suggest that we use a `<Fragment>` React element and put direct HTML in here instead. That will make l10n easier too, as not all languages express this meaning in this exact order of words.

Reply:
It's a temporary UI. Can't we leave it as the current state?

Reply:
I am not supposed to rule on that. Ask the PM or designer.

Reply:
@neruthes

Reply:
Since it is just a rush demo, I would prefer to conclude things in #70 instead of making issue-specific changes over the current jam.

Reply:
I feel no need to rush to refactor i18n for now. If you find changing i18n framework after delivering #70 will be greatly harder, we may further investigate. Current priority is #287 and #70.